tests/test-extensions/metadata-extension/src/main/java/org/mule/test/metadata/extension/resolver/TestInputResolver.java

```diff
@@ -11,9 +11,12 @@
 import org.mule.runtime.api.metadata.MetadataContext;
 import org.mule.runtime.api.metadata.MetadataResolvingException;
 import org.mule.runtime.api.metadata.resolving.InputTypeResolver;
+import org.mule.runtime.api.metadata.resolving.MetadataComponent;
 
 public class TestInputResolver implements InputTypeResolver<String> {
 
+  public static String INPUT_RESOLVER_NAME = MetadataComponent.INPUT.name();
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make this "public static INPUT_RESOLVER_NAME" field final ([squid:S1444](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1444))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7668&issues=AWjdfEJ1otIXnmsUGfbJ&open=AWjdfEJ1otIXnmsUGfbJ)

[ISSUE_KEY]: AWjdfEJ1otIXnmsUGfbJ